index.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable semi */
+const bookTitle = document.getElementById('book-title');
+const authorName = document.getElementById('book-author');
+const button = document.getElementById('submit');
+const list = document.querySelector('#books-ul');
+// const array = [];
+// let counter = 0;
+// let delcounter = 0;
```

Comment:
- [ ] I would like to recommend you remove all the commented code from your codebase in order to keep your code clean :smile: